### DISCUSSION TITLE
GRASS Start-Up Issue (New User)

### DISCUSSION THREAD

**Discussion by mcbburton** [OSGeo/grass#2897] [Q&A]
I apologize for the basic terms; I am a new user as of this semester.  We are utilizing GRASS 8.2.
I have had no difficulties using GRASS to date for our projects.
I last logged in about 4 days ago.
I started the GRASS command today, and am receiving a value error message.
Unfortunately I am not sure how to get past this issue.  It appears to reference my map sets, which I have not made any changes to since I last used the program. 

Any assistance is appreciated.
![image](https://user-images.githubusercontent.com/128253364/226129737-1f4421ba-fb14-4912-9cbd-eaf6a52bcbd5.png)

I am running Windows 10.

**Comment by dnewcomb** [comment]
I would try reinstalling with 8.2.1 first
